client/server/bundler/sections-loader.js

```diff
@@ -55,12 +55,29 @@ function printSectionsAndPaths( sections ) {
 	}
 }
 
+function filterSections( sections ) {
+	const bundleEnv = config( 'env' );
+	if ( 'development' !== bundleEnv ) {
+		return sections;
+	}
+
+	const activeSections = config( 'sections' );
+	const byDefaultEnableSection = config( 'enable_all_sections' );
+
+	return sections.filter( section => {
+		if ( activeSections && typeof activeSections[ section.name ] !== 'undefined' ) {
```

Comment:
Not sure about this but shouldn't the comparison be `activeSections[ section.name ] !== undefined` instead of `activeSections[ section.name ] !== 'undefined'`? Can the name of a section be the `'undefined'` string?

Reply:
@rcanepa - As it's using [`typeof`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof), that returns a string of the type of operator, not the value of the operator.

Reply:
What @ChaosExAnima said :)